src/EnergyPlus/DaylightingManager.cc

```diff
@@ -3330,16 +3227,25 @@ void FigureDayltgCoeffsAtPointsForSunPosition(
     // Rob/TH - Not sure whether this call is necessary for interior zones with interior windows only.
     //  new code would be -
     // IF (LSHCAL == 1 .AND. ExtWinType /= AdjZoneExtWin) CALL DayltgInterReflectedIllum(ISunPos,IHR,ZoneNum,IWin2)
+    // TODO MJW: This may be getting repeated if there is more than one daylighting control in an enclosure
```

Comment:
"more than one daylighting control in an enclosure" ?
Isn't it space or zone for daylight controls?
I can't see the whole picture here but where is zoneNum used? There it is, 3463. Is it worth saving zoneNum for one use? Picky but to the overall point that if not used more than twice, don't save it.

Reply:
I know, zones/spaces/enclosures/controls make heads spin. I've added some comments at the top of the PR to help and will need to beef up the documentation.